### ISSUE TITLE
No such file or directory @ utime_internal

### ISSUE THREAD

**Issue by lencioni** [sds/overcommit#55]
```
No such file or directory @ utime_internal - /path/to/repo/node_modules/.bin/bower
/Users/joe/.rbenv/versions/2.1.0/lib/ruby/gems/2.1.0/gems/overcommit-0.9.0/lib/overcommit/hook_context/pre_commit.rb:102:in `utime'
/Users/joe/.rbenv/versions/2.1.0/lib/ruby/gems/2.1.0/gems/overcommit-0.9.0/lib/overcommit/hook_context/pre_commit.rb:102:in `block in restore_modified_times'
/Users/joe/.rbenv/versions/2.1.0/lib/ruby/gems/2.1.0/gems/overcommit-0.9.0/lib/overcommit/hook_context/pre_commit.rb:100:in `each'
/Users/joe/.rbenv/versions/2.1.0/lib/ruby/gems/2.1.0/gems/overcommit-0.9.0/lib/overcommit/hook_context/pre_commit.rb:100:in `restore_modified_times'
/Users/joe/.rbenv/versions/2.1.0/lib/ruby/gems/2.1.0/gems/overcommit-0.9.0/lib/overcommit/hook_context/pre_commit.rb:34:in `cleanup_environment'
/Users/joe/.rbenv/versions/2.1.0/lib/ruby/gems/2.1.0/gems/overcommit-0.9.0/lib/overcommit/hook_runner.rb:25:in `ensure in run'
/Users/joe/.rbenv/versions/2.1.0/lib/ruby/gems/2.1.0/gems/overcommit-0.9.0/lib/overcommit/hook_runner.rb:25:in `run'
.git/hooks/pre-commit:70:in `<main>'
Report this bug at https://github.com/causes/overcommit/issues
```

**Comment by trotzig** [comment]
I just ran into this on version 0.24.0. I had a fairly large git index about to be committed. I think I hit Ctrl-C twice, because the first one didn't seem to have any effect (logs kept coming in the console). Here's some output:

```bash
⨠ g amend                                                                                             │
^C                                                                                                    │
→ exit status: 130                                                                                    │
Running pre-commit hooks                                                                              │
henrictrotzig@Henrics-MacBook-Air-13:~/brigade 92838002-contacts-platform-platform                    ├
⨠ Analyzing with ESLint........................................[EsLint]                               │
→ exit status: 1                                                                                      │
henrictrotzig@Henrics-MacBook-Air-13:~/brigade 92838002-contacts-platform-platform                    │
⨠ WARNING                                                                                             │
Warnings on modified lines:                                                                           │
/Users/henrictrotzig/brigade/spec/javascripts/components/ask_supporters_line_spec.jsx: line 23, col 0,│
 Warning - Line 23 exceeds the maximum line length of 80. (max-len)                                   │
/Users/henrictrotzig/brigade/spec/javascripts/components/position_reveal_friends_zero_state_spec.jsx: │
line 17, col 0, Warning - Line 17 exceeds the maximum line length of 80. (max-len)                    │
Warnings on lines you didn't modify:                                                                  │
/Users/henrictrotzig/brigade/app/assets/javascripts/intl_data/messages.js: line 944, col 0, Warning - │
Line 944 exceeds the maximum line length of 80. (max-len)                                             │
/Users/henrictrotzig/brigade/spec/javascripts/components/position_reveal_friends_spec.jsx: line 112, c│
ol 0, Warning - Line 112 exceeds the maximum line length of 80. (max-len)                             │
                                                                                                      │
Analyzing with scss-lint...................................[ScssLint]                                 │
→ exit status: 1                                                                                      │
```

After this happened, my changes were gone. They weren't in the git stash either.

**Comment by trotzig** [comment]
I managed to recover my changes with a bit of reflogging. I ran this command:

```bash
gitk --all $( git fsck --no-reflog | awk '/dangling commit/ {print $3}' )`
```

Which showed me all dangling commits. In that list I found a commit with the following commit message: `index on 92838002-contacts-platform-platform: 92f4bdd Fix styling of card stack`. All my changes were in there. 

Here's where I found that command:
http://stackoverflow.com/questions/89332/recover-dropped-stash-in-git
I realize I haven't given any good information about how to reproduce or track the issue down. But perhaps some of what I experienced can help others running into the same thing.
